_schemaV2.graphql

```diff
@@ -4685,6 +4685,37 @@ type Location {
   summary: String
 }
 
+# A connection to a list of items.
+type LocationConnection implements LocationsConnectionInterface {
```

Comment:
Looks like the schema needs to be re-generated to remove these interfaces?

Reply:
Regenerated.